L1Trigger/TrackFindingTracklet/python/l1tTTTracksFromTrackletEmulation_cfi.py

```diff
@@ -36,6 +36,6 @@
                                                tableTEDFile = cms.FileInPath('L1Trigger/TrackFindingTracklet/data/table_TED/table_TED_D1PHIA1_D2PHIA1.txt'),
                                                tableTREFile = cms.FileInPath('L1Trigger/TrackFindingTracklet/data/table_TRE/table_TRE_D1AD2A_1.txt'),
                                                # Quality Flag and Quality params
-                                               TrackQuality = cms.bool(False),
+                                               TrackQuality = cms.bool(True),
```

Comment:
This change will affect all downstream users of the track finding emulation so this change should be confirmed with the track finding group that they do want the default track finding to have the track quality turned on

Reply:
Maybe I could leave the default as is but make a note if my vertexing config that this should be turned on if using the displaced vertexing?

Reply:
Yes I am happy with that solution

Reply:
Addressed in https://github.com/cms-sw/cmssw/pull/45672/commits/276ecb95fae3fd47e9d4eeb7ab7c722bd846d158

Reply:
hmm strangely when I am looking through the code under `Files Changes` with the latest commit `b5b8115` I still see it as `TrackQuality = cms.bool(True)` can you confirm this is now False please? And L23 is True. (In fact can you make sure the changes you made in commit https://github.com/cms-sw/cmssw/commit/276ecb95fae3fd47e9d4eeb7ab7c722bd846d158 were successful?